File: spec/globals.js
function assert(b) {
    if(!b) {
        throw "assertion failed";
    }
}

function test() {
    assert(typeof(global_int) == "number");
    assert(typeof(global_array) == "object");
    assert(typeof(global_boolean) == "boolean");
    assert(typeof(global_float) == "number");
    assert(typeof(global_int) == "number");
    assert(typeof(global_string) == "string");
    return "ok";
}

File: spec/globals.lua
function assert(b)
    if not b then
        error("assertion failed")
    end
end

function test()
    assert(type(global_int) == "number")
    assert(type(global_array) == "table")
    assert(type(global_boolean) == "boolean")
    assert(type(global_float) == "number")
    assert(type(global_string) == "string")
    return "ok"
end
